Install Mizu in a dedicated namespace

nimrod-up9 opened the pull request:
1. Install Mizu resources (tappers daemonset, controller pod, collector service, serviceacount) in a dedicated "mizu" namespace.
2. When removing Mizu resources, delete the namespace instead of deleting each resource individually.
3. When removing Mizu resources, remove clusterrole and clusterrolebinding.
4. When removing Mizu resources, wait for resources to be deleted before exiting.
5. A second ctrl-c will skip the wait.

IgorGov commented on cli/mizu/consts.go:
```diff
@@ -9,11 +9,14 @@ var (
-	AggregatorPodName   = "mizu-collector"
-	TapperPodName       = "mizu-tapper"
-	K8sAllNamespaces    = ""
+	AggregatorPodName      = "mizu-collector"
```
CollectorPodName

IgorGov commented on cli/kubernetes/provider.go:
```diff
@@ -71,7 +113,16 @@ func (provider *Provider) GetPodWatcher(ctx context.Context, namespace string) w
+	return provider.clientSet.CoreV1().Namespaces().Create(ctx, namespaceSpec, metav1.CreateOptions{})
+}
+
+func (provider *Provider) CreateMizuAggregatorPod(ctx context.Context, namespace string, podName string, podImage string, serviceAccountName string, mizuApiFilteringOptions *shared.TrafficFilteringOptions) (*core.Pod, error) {
```
Aggregator => Collector

ksudhir007 commented:
Thank you very much for this! Can you also please update README permissions sections as part of this pull request ? It will help others give an overview on the RBAC permissions in shared cluster environment.

nimrod-up9 commented:
> Thank you very much for this! Can you also please update README permissions sections as part of this pull request ? It will help others give an overview on the RBAC permissions in shared cluster environment.

Good idea! Updated the README and added example resource yamls for the required clusterroles.
